.gitignore

```diff
@@ -1,3 +1,4 @@
-_build
+/_build/
+/_opam/
```

Comment:
What's the intent here? Only to match _build and _opam in the top directory of the project?

Reply:
BTW, I'd suggest not changing the .gitignore in the same commit...

Reply:
Exactly, only match if they are directories and only match if they are on the top of the repository. I don't like random (though unlikely) surprises of `_opam` and `_build` missing due to overzealous ignore rules, when there are exactly two cases that I want to match.

In fact the `*.install` rule could also be reformulated to be `/*.install`…

Reply:
I've split the `.gitignore` change into its own commit.

Reply:
It looks like it is still here?

Reply:
It is split out in its own commit, as you requested. I've also took the liberty to amend the original commit to include the revised description, d3a501566b3c673bc67c478b1be980d4a705c50e so the `.gitignore` commit is in 8853750d2f745b826801347ebdbd38ce635b94e9.

Reply:
I meant it should be merged in a distinct PR. (One PR, one concern.)

But it's not that big a deal.

@NathanReb as the new maintainer, are you happy with this? If so, please merge it!

Reply:
I think splitting the commit is good, browsing the git history makes the intent pretty clear, which is what matters the most. Let's not put too much of a burden on @Leonidas-from-XIV !